packages/redux/src/products/selectors/listings.ts

```diff
@@ -760,3 +761,49 @@ export const getProductListingFacetGroups = createSelector(
   state: StoreState,
   hash?: string | number | null,
 ) => FacetGroupDenormalized[] | undefined;
+
+/**
+ * Returns all facets by parent type.
+ *
+ * @param state - Application state.
+ * @param facetGroupType - Facet group type to find.
+ *
+ * @returns Array with all facets content requested by group type.
+ */
+export const getFacetsByGroupType = (
+  state: StoreState,
+  facetGroupType: FacetGroup['type'],
+) =>
+  Object.values(getFacets(state) || []).filter(
+    facet => facet?.groupType === facetGroupType,
+  );
+
+/**
+ * Returns the listing facets error.
+ *
+ * @param state - Application state.
+ *
+ * @returns Listing Facets error.
+ */
+export const getListingFacetsError = (state: StoreState) =>
+  getListingFacetsState((state.products as ProductsState).lists).error;
+
+/**
+ * Returns the result of the listing facets.
```

Comment:
```suggestion
 * Returns the result of the fetch listing facets request.
```

The same for other selectors

Reply:
Fixed!